src/main/features/claim/views/eligibility/government-department.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, radioGroup, saveAndContinueButton %}
+
+{% set title = t('Govornment department | Money claim') %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Are you claiming against a government department?') }}</h1>
+      <p>
+        {{ t('You can’t use this service to make a claim against a government department such as HM Revenue and Customs.') }}
+      </p>
+
+      {% set name = 'governmentDepartment' %}
+
+      <form novalidate method="post" class="analytics-submit-event-trigger"
+            data-event-action="Eligibility: Against government department option"
+            data-event-label-from="{{ name }}">
+        {{ csrfProtection(csrf) }}
+
+        {{
+          radioGroup(
+            'Are you claiming against a government department?',
+            name,
+            [
+              {'label': 'Yes', value: 'yes', checked: (form.valueFor(name) === YesNoOption.YES)},
```

Comment:
Shouldn't the value be take from `YesNoOption` as well?

Reply:
Yup